charts/ccd-data-store-api/values.preview.template.yaml

```diff
@@ -27,8 +27,10 @@ java:
     DATA_STORE_DB_PASSWORD: "{{ .Values.postgresql.auth.password}}"
     DATA_STORE_DB_OPTIONS: "?stringtype=unspecified"
     DATA_STORE_DB_MAX_POOL_SIZE: 10
-    DEFINITION_STORE_HOST: http://ccd-definition-store-api-pr-575-java/
-    USER_PROFILE_HOST: http://ccd-user-profile-api-pr-399-java/
+    # TODO: remove TEMPORARY OVERRIDE to test ElasticSearch, reinstate: DEFINITION_STORE_HOST: http://ccd-definition-store-api-pr-575-java/
+    DEFINITION_STORE_HOST: http://ccd-definition-store-api-pr-1279-java/
```

Comment:
This should be reveresed if final test-def version changes are merged to `master`.